Gigs/Gigs/Application/GigController.swift

```diff
@@ -147,17 +147,17 @@ class GigController {
             
             do {
                 let gigNames = try decoder.decode([Gig].self, from: data)
-                completion(.success(gigNames))
-                self.gigs.append(contentsOf: gigNames)
+                completion(.success(true))
+                self.gigs = gigNames
             } catch {
                 completion(.failure(.decodingError))
             }
             
-    }.resume()
+        }.resume()
         
     }
     
-    func createGigs(with gig: Gig, completion: @escaping (Result<Gig, NetworkError>) -> Void ){
+    func createGigs(with gig: Gig, completion: @escaping (Result<Bool, NetworkError>) -> Void ){
```

Comment:
Again, you aren't going to need to use the gig in your callback, so no need to pass it through